lib/typescript/httpclient/model/Channel.ts

```diff
@@ -1,8 +1,14 @@
+import {Metadata} from './Metadata';
+
+export type ChannelMetadata = Metadata & {
+  name: string;
+  image_url?: string;
```

Comment:
Shouldn't this be `imageUrl` ? We decided to have camelCase for every interface that exposes a structure to the outside of the http client lib

Reply:
Yes I will change this